sklearn/model_selection/_validation.py

```diff
@@ -455,29 +499,37 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     return_estimator : boolean, optional, default: False
         Whether to return the fitted estimator.
 
+    return_fit_failed : bool, default=False
```

Comment:
This is a private function. There's no need to add a parameter for this kind of change.